index.js

```diff
@@ -97,6 +97,11 @@ class PQueue {
 			}
 		});
 	}
+
+	clear() {
+		this.queue._queue = [];
```

Comment:
@SamVerschueren there could be come logic attached to `dequeue` method and we will not trigger it, but I'm not sure that it will do right thing, because we not call extracted function. Beside this – none.